src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/client/DeliusClientTest.kt

```diff
@@ -25,7 +25,8 @@ class DeliusClientTest : IntegrationTestBase() {
 
   @Test
   fun `find by crn returns null when not found`() {
-    val response = deliusClient.findByCrn("X123456")
+    findByCrnNotFound(crn = "X654321")
```

Comment:
My understanding is that wiremock should be returning 404 by default if it receives an unmapped request. Was this not the case? I'm OK with still adding this code (either that or adding a comment explicitly stating the reason why we don't make any wiremock config calls), but I'm wondering if it isn't doing so (and why).

Reply:
This is to be explicit for this test case as it states the desired behaviour rather than relying on a default. I mostly did this for my own confidence when exploring how the client behaved and thought "might as well leave that there as it self documents".